package.json

```diff
@@ -8,8 +8,9 @@
     "build": "webpack --config webpack.config.js",
     "test": "react-scripts test",
     "test:coverage": "CI=true npm test -- --env=jsdom --coverage",
-    "coverage": "react-scripts test --coverage a"
-      },
+    "test:coveralls": "jest --coverage && coveralls < coverage/lcov.info",
```

Comment:
You don't need to create new scripts, because we already have test and coverage scripts. What you can add is `./coverage/lcov.info | coveralls` in one of those scripts. But it's better if it is coverage. Because it will upload coverage info after the coverage has been done.